scaffolding-chef-inspec/lib/scaffolding.ps1

```diff
@@ -0,0 +1,176 @@
+  Start-Sleep -Seconds `$SPLAY_DURATION
+  Start-Sleep -Seconds `$env:CFG_INTERVAL
+  Invoke-Inspec
+}
+"@
+}
+
+function Invoke-DefaultBuild {
+    Copy-Item $PLAN_CONTEXT/../* $HAB_CACHE_SRC_PATH/$pkg_dirname -Recurse -Force
+    inspec archive "$HAB_CACHE_SRC_PATH/$pkg_dirname" `
+                   --overwrite `
+                   -o "$HAB_CACHE_SRC_PATH/$pkg_dirname/$pkg_name-$pkg_version.tar.gz"
+}
+
+function Invoke-DefaultInstall {
+    Copy-Item "$HAB_CACHE_SRC_PATH/$pkg_dirname/$pkg_name-$pkg_version.tar.gz" "$pkg_prefix"
+    New-Item -Type Directory $pkg_prefix/config
+
+    Write-BuildLine "Generating config/inspec_exec_config.json"
+    Add-Content -Path "$pkg_prefix/config/inspec_exec_config.json" -Value @"
+{
+    "target_id": "{{ sys.member_id }}",
+    "reporter": {
+        "cli": {
+          "stdout": {{cfg.report_to_stdout}}
+        },
+        "json": {
+          "file": "{{pkg.svc_path}}/logs/inspec_last_run.json"
+        }{{#if cfg.automate.enable ~}},
+        "automate" : {
+          "url": "{{cfg.automate.server_url}}/data-collector/v0/",
+          "token": "{{cfg.automate.token}}",
+          "node_name": "{{ sys.hostname }}",
+          "verify_ssl": false
+        }{{/if ~}}
+    }
+    {{#if cfg.automate.enable }},
+    "compliance": {
+      "server" : "{{cfg.automate.server_url}}",
+      "token" : "{{cfg.automate.token}}",
+      "user" : "{{cfg.automate.user}}",
+      "insecure" : true,
+      "ent" : "automate"
+    }{{/if }}
+}
+"@
+
+    Write-BuildLine "Generating config/cli_only.json"
+    Add-Content -Path "$pkg_prefix/config/cli_only.json" -Value @"
+{
+    "reporter": {
+        "cli" : {
+            "stdout" : true
+        }
+    }
+}
+"@
+
+    Write-BuildLine "Generating Chef Habitat configuration default.toml"
+    Add-Content -Path "$pkg_prefix/default.toml" -Value @"
+# You must accept the Chef License to use this software: https://www.chef.io/end-user-license-agreement/
+# Change [chef_license] from acceptance = "undefined" to acceptance = "accept-no-persist" if you agree to the license.
+
+interval = 1800
+splay = 1800
+splay_first_run = 0
+log_level = 'warn'
+report_to_stdout = true
+
+[chef_license]
+acceptance = "undefined"
+
+[automate]
+enable = false
+server_url = 'https://<automate_url>'
+token = '<automate_token>'
+user = '<automate_user>'
+"@
+}
+
+function Invoke-ComplianceLogin {
+    if (!$COMPLIANCE_CREDS){ 
+        Write-BuildLine "ERROR: Please preform an 'inspec compliance login' and set"
+        Write-BuildLine " `$HAB_STUDIO_SECRET_COMPLIANCE_CREDS to the contents of"
+        Write-BuildLine " '~/.inspec/compliance/config.json'"
+        exit 1
+    }
+
+    $creds = Get-Content $COMPLIANCE_CREDS | ConvertFrom-Json
+
+    inspec compliance login --insecure $creds.insecure `
+                            --user $creds.user `
+                            --token $creds.token `
+                            "$($creds.server)/api/v0"
```

Comment:
`/v0` should have a trailing slash `/v0/`

Reply:
Oh damn, does `ConvertFrom-Json` give us a native jq-like powershell object? That's awesome.

Reply:
Yep I will add the trailing slash good catch